go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/miekg/dns
 go 1.12
 
 require (
-	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
-	golang.org/x/net v0.0.0-20190923162816-aa69164e4478
-	golang.org/x/sync v0.0.0-20190423024810-112230192c58
-	golang.org/x/sys v0.0.0-20190924154521-2837fb4f24fe
-	golang.org/x/tools v0.0.0-20191216052735-49a3e744a425 // indirect
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
```

Comment:
The changes to go.mod and go.sum are unrelated.

Reply:
it should be fixed in #e62bf83

Reply:
I'm still seeing the go.mod and go.sum changes in the PR diff.